toolkit/offchain/src/init_governance/mod.rs

```diff
@@ -113,3 +113,39 @@ pub async fn run_init_governance<T: QueryLedgerState + Transactions + QueryNetwo
 
 	Ok(result.transaction)
 }
+
+pub async fn get_governance_utxo<T: QueryLedgerState + Transactions + QueryNetwork>(
+	genesis_utxo: UtxoId,
+	client: &T,
+) -> anyhow::Result<OgmiosUtxo> {
+	let network = client.shelley_genesis_configuration().await?.network;
+
+	let version_oracle_validator =
+		PlutusScript::from_wrapped_cbor(raw_scripts::VERSION_ORACLE_VALIDATOR, LanguageKind::PlutusV2)?
+			.apply_uplc_data(datum_to_uplc_plutus_data(&genesis_utxo))?;
+
+	let validator_address = version_oracle_validator.address_bech32(NetworkTypeExt::to_csl(&network))?;
+
+	let version_oracle_policy =
+		PlutusScript::from_wrapped_cbor(raw_scripts::VERSION_ORACLE_POLICY, LanguageKind::PlutusV2)?
+			.apply_uplc_data(datum_to_uplc_plutus_data(&genesis_utxo))?
+			.apply_uplc_data(version_oracle_validator.address_data(NetworkTypeExt::to_csl(&network))?)?;
```

Comment:
Lines above belong to scripts_data.rs. If the data required below is missing, please add to the output there or alternatively expose `pub(crate)` functions there.